src/bp/ui-shared/src/index.tsx

```diff
@@ -13,7 +16,9 @@ import TreeView from './TreeView'
 
 exports.Dialog = { Wrapper, Footer, Body }
 exports.Dropdown = Dropdown
+exports.EmptyState = EmptyState
 exports.MainContainer = MainContainer
+exports.MainContent = { Header, Wrapper: MainContentWrapper }
```

Comment:
I think we should keep the main index file as clean as possible, what do you think of exporting both components in `MainContent/index.tsx` instead, and simply exporting `exports.MainContent = MainContent` here?